### ISSUE TITLE
reference to previous argument via incipit

### ISSUE THREAD

**Issue by jeffreycwitt** [lombardpress/lombardpress-schema#131]
I'm struggling with the proper encoding of passages like the following, whether the author is referencing a prior argument with a small quote from the beginning of the argument, as follows.

Et quando dicitur **quomodo doctor** etc., dicitur quod forte eorum pertinacia 

Usually this is an argument within an argument, and therefore embedded with a larger paragraph.

I see three possibilities for encoding this, `<mentioned>quomodo doctor</mentioned`, `<quote>mentioned<quote>`, `<ref><seg type="incipit">quomodo doctor</seg> etc.</ref>`

(this example can be found in the text of http://scta.info/resource/pg-b1q11)

Is this really a "quotation"? Would we want this showing up in the quotation app?
`Mentioned` would generally wrap this in single quotation marks, which people often do, but this would be to treat it the same way we treat texts like "li <mentioned>dicere<mentioned>" and these don't seem like parallel cases. 

It seems kind of like a reference, where part of the reference is an incipit.

`@target` attribute on the reference could point to an identifier for the entire argument. This might require offset markup to identify the entire text range of the argument. 

It might also be nice to have a similar attribute available for marking the precise text range that the incipit refers to. So for example <seg type="incipit" target="paragraphid@13-15"> 

Such a target would require a way to reference overlapping ranges of texts within a paragraph, which is a larger issue. 

For the moment, I think I would vote for treating this as a reference with a `seg@type` incipit.

**Comment by jeffreycwitt** [comment]
But what about case like the following

"when Paul says 'love' he means"

What is the status of "love" in this case.?

The structure is similar to the above, 
but I would be tempted to you use `mentioned` here and not `ref`

This happens a lot in commentaries. 

An author will quote a bible verse, and then below will pick out individual words and provide interpretations. 

In these cases, they are words from a "quote" but it seems silly to treat each word as a separate quote. 

I would be strongly inclined to use `mentioned` here. 

Again perhaps with the ability to add a target identifier like <mentioned target="#quoteid@5">love</mentioned>. Here we can connect the word being discussed to the quote containing the word by the word position (position 5) of the word within the quote.

But how different is this from the above?

**Comment by jeffreycwitt** [comment]
I'm really going back and forward on this. 

Today, I worked a text from Peter of Candia that is replete with the following pattern

```xml
Ad secundam, cum dicebatur, <quote>divina essentia non habet
                  distinctam notitiam futuri contingentis, ergo quaestio falsa</quote>, concedo
                  consequentiam et nego antecedens. Et ad probationem, cum dicitur, <quote>quia si
                  haberet notitiam futuri contingentis, cum a variatione scibilis sequatur variatio
                  notitiae sive scientiae, sequitur quod cum futurum contingens sit variabile, quod
                  et similiter notitia divina esset variabilis</quote>, hic dico
```
He does this 20 or so times. 

he is responding to the arguments of Thomas Bradwardine, but he has introduced and quoted the passages above. Now providing nuanced interpretations of individual sentences or parts of the argument. 

Further "ad secundum" could be both a reference to the second argument introduced by Candia above but also to the second argument introduced by Bradwardine in his text. In fact, it is in a way a reference to both. Same goes for "ad probationem". Thus I would be inclined to something like this: 

```xml
<ref>Ad secundam</ref>, 
cum dicebatur, 
<quote>
divina essentia non habet
distinctam notitiam futuri contingentis, ergo quaestio falsa</quote>, 
concedo consequentiam et nego antecedens. 
Et 
<ref>ad probationem<ref>, 
cum dicitur, 
<quote>quia si
haberet notitiam futuri contingentis, cum a variatione scibilis sequatur variatio
notitiae sive scientiae, sequitur quod cum futurum contingens sit variabile, quod
et similiter notitia divina esset variabilis</quote>, 
hic dico...
```

This conflicts with my earlier inclination to treat 

"Et quando dicitur quomodo doctor etc., dicitur quod forte eorum pertinacia"

as a reference with an "incipit".

So I'm not sure what to do.

**Comment by stenskjaer** [comment]
In cross references within a commentary I have tended to use `<mentioned>` and `<quote>` so far without being systematic. In a way it is a quote, just of the author himself. I see how this of course may have the unfortunate effect of putting the quote in a list of quotes where you expect external and not internal quotes. But I'm not sure that is enough reason to modify the semantics.

On the "love" example: That is, to my mind, clearly a `<mentioned>`. 

I think using `quote` for self-quotes or internal quotes is natural because that also makes it fit into the model for referencing in the apparatus fontium with relatively few modifications:

```xml
Et quando dicitur 
<cit>
    <quote type="internal" target="#id">quomodo doctor etc.</quote>
</cit>
, dicitur quod forte eorum pertinacia
```

So in stead of the `<bibl>` the reference is made to the relevant part of the text when `@type="internal"`. Of course the problems with exact references or references of more than one paragraph still remain.

**Comment by jeffreycwitt** [comment]
One worry I have with the idea of "internal", is that "internal" is kind of a relative concept. 
Internal to the present file? Internal to this section of the work? Or internal to the entire work which could be thousands of pages. 

Since we have the ability to combine and separate individual TEI files depending on need, the concept of internal breaks down as soon as the composition of the file changes. 

This is why for `quote@source` I always use the full SCTA url. 

`<quote source="http://scta.info/resource/paragraph_whatever>Quote</quote>`

Then when I run the LaTeX build for example, internal cross references are logged in the apparatus fontium if the referenced passage is included in the same typsetting, and references are treated as external when the referenced passage is not present. 

In other words, I think references vs. cross-references is a presentation specific concept, so I compute this when the presentation is created. 

(I can show you an example of how I'm doing this with LaTeX if that will be helpful). 

But I think I we're both agreed that there is some ambiguity about when to use `mention` vs `quote` when an author is referring to an argument previous given. I too have been going back and forth, and thus, have likely been quite inconsistent.

It will be useful when we get to together to create a report of all currently tagged instances and see what kind of patterns of existing practice already exist. Then we can try to systematize.

**Comment by jeffreycwitt** [comment]
In the original example "quomodo doctor would be a `<quote type="lemma">`.
